.eslintrc.json

```diff
@@ -26,6 +26,16 @@
     "rules": {
         "react/prop-types": "off",
         "react/jsx-filename-extension": "off",
-        "no-console": "error"
+        "no-console": "error",
+        "jsx-a11y/label-has-associated-control": [
+            "error",
+            {
+                "labelComponents": [],
+                "labelAttributes": [],
+                "controlComponents": [],
+                "assert": "either",
+                "depth": 25
+            }
+        ]
```

Comment:
There's more info about this rule here:
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md
The default is to have:

```json
"assert": "both"
```

This isn't compatible with having a visually hidden label.